src/test/lib/concurrency/stress_test.cpp

```diff
@@ -587,4 +587,105 @@ TEST_F(StressTest, OperatorRegistration) {
   }
 }
 
+// For cases where issues usually arise early, we run the test multiple times.
+class StressTestMultipleRuns : public StressTest, public ::testing::WithParamInterface<int> {};
+
+INSTANTIATE_TEST_SUITE_P(StressTest, StressTestMultipleRuns, testing::Range(0, 10),
+                         [](const testing::TestParamInfo<StressTestMultipleRuns::ParamType>& info) {
+                           return "loop" + std::to_string(info.param);
+                         });
+
+/**
+ * Test to verify that rollbacking Insert operations does not lead to multiple (outdated) rows being visible. This issue
```

Comment:
rolling back?

Reply:
Hehe, promise me that you have never seen this!

Reply:
🤫